modules/developer_manual/pages/testing/drone.adoc

```diff
@@ -0,0 +1,85 @@
+= Drone Testing
+:toc: right
+
+:drone-cli-url: https://github.com/drone/drone-cli
+:smashbox-url: https://github.com/owncloud-docker/smashbox
+
+== Introduction
+
+This document will show you how to run drone tests locally.
+
+== Setup Drone
+
+- Download the {drone-cli-url}[command line client] for the Drone Continuous Integration Server
+- Have a local docker available
+- Look at file `.drone.yml` for possible values or the variables as they appear in the CI drone runs
+- Command: `PHP_VERSION=7.1 DB_TYPE=postgres TEST_SUITE=phpunit drone exec`
```

Comment:
[PHP 7.2 is already no longer supported](https://www.php.net/eol.php), if in any way possible I would try to use a supported PHP version in a new document.

Reply:
changed to php 7.4